Sources/App/Models/FundingLink.swift

```diff
@@ -32,6 +33,7 @@ struct FundingLink: Codable, Equatable {
 
     var platform: Platform
     var url: String
+    var urlLabel: String
```

Comment:
Nit: I think we should call this `rootDomain` and make it

```swift
var rootDomain: String?
```

and then to the `?? URL(string: url)?.host ?? url` on the package page view side. That way we don't store view related data in the db.

Reply:
Or we could, and this may be better, push the label into the `customUrl` case. Let me experiment with that.